pkgs/by-name/li/linux-pam/package.nix

```diff
@@ -21,7 +21,7 @@
   docbook_xsl_ns,
   nix-update-script,
   withLogind ? lib.meta.availableOn stdenv.hostPlatform systemdLibs,
-  withAudit ? lib.meta.availableOn stdenv.hostPlatform audit,
+  withAudit ? lib.meta.availableOn stdenv.hostPlatform audit && !stdenv.buildPlatform.isDarwin,
```

Comment:
Alternatively if want to be more general:
```
withAudit ? lib.meta.availableOn stdenv.hostPlatform audit && stdenv.buildPlatform.isLinux
```

Reply:
```suggestion
  withAudit ? lib.meta.availableOn stdenv.hostPlatform audit && lib.meta.availableOn stdenv.buildPlatform linuxHeaders ,
```
maybe?